client/src/components/screens/Patient/components/PatientHeader/components/ProbandHeader.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import intl from 'react-intl-universal';
+import { Col, Tag, Typography } from 'antd';
+import { ParsedPatientData } from 'helpers/providers/types';
+
+import FemaleIcon from 'components/Assets/Icons/FemaleIcon'
+import MaleIcon from 'components/Assets/Icons/MaleIcon';
+import UnknowGenderIcon from 'components/Assets/Icons/UnknowGenderIcon'
+
+import '../styles.scss';
+  
+enum Gender {
+MALE = 'male',
+FEMALE = 'female',
+UNKNOWN = 'unknown',
+}  
+
+const getGenderIcon = (gender: string) => {
```

Comment:
Je trouve les switch un peu plus propre, mais c'est mon avis:

```suggestion
const getGenderIcon = (gender: string) => {
  switch (gender) {
    case Gender.MALE:
      return <MaleIcon className="customIcon" height="22" viewBox="0 0 22 22" width="22"/>;
    case Gender.FEMALE:
      return <FemaleIcon className="customIcon" height="22" viewBox="0 0 22 22" width="22" />;
    default:
      return <UnknowGenderIcon className="customIcon" height="23" viewBox="0 0 22 23" width="22" />
  }
}
```